samples/cpp/CMakeLists.txt

```diff
@@ -21,6 +21,36 @@ project(Samples
         HOMEPAGE_URL "https://docs.openvino.ai/2025/get-started/learn-openvino/openvino-samples.html"
         LANGUAGES C CXX)
 
+if(NOT DEFINED OPENVINO_STD_FS_LIB)
```

Comment:
is it supposed that all users applications have to write the same code to enable their samples / apps to work correctly?

I suppose we need to make `OPENVINO_STD_FS_LIB` to be a public dependency for OpenVINO, so it's automatically added to user apps when OpenVINO depends on it.

Reply:
I've added explicit linking for specific compilers and version according cppreference.com

Reply:
great, looks much better IMO